vep/README.md

```diff
@@ -0,0 +1,74 @@
+
+Script is parametrised with `__VEP_VERSION__`, so pass it through `sed` when copying to the bucket:
+
+```shell
+cat dataproc-init.sh | sed "s/__VEP_VERSION__/${VEP_VERSION}/g" | gsutil cp - gs://cpg-reference/vep/${VEP_VERSION}/dataproc/init.sh
+```
+
+2. Similarly, the JSON config script for dataproc [dataproc-config.json](dataproc-config.json) is also copied from the [Hail codebase](https://github.com/hail-is/hail/blob/cc0a051740f4de08408e6a2094ffcb1c3158ee9c/hail/python/hailtop/hailctl/hdinsight/resources/vep-GRCh38.json) and modified to reflect the newer VEP versions, and additionally has `mane_select:String,mane_plus_clinical:String` in schema to load MANE IDs that are supported by modern versions of VEP.
+
+The config is parametrised with `__VEP_VERSION__`, so pass it through `sed` when copying to the bucket:
+
+```sh
+cat dataproc-config.json | sed "s/__VEP_VERSION__/${VEP_VERSION}/g" | gsutil cp - gs://cpg-reference/vep/${VEP_VERSION}/dataproc/config.json
+```
+
+We also have a separate version of the config `dataproc-config-no-exac.json` for older VEP versions (before v88). In this version, ExAC fields are excluded: they used to contain non-numerical format for ExAC frequencies, e.g. `"exac_adj_maf":"-:0.2934,-:8.292e-06", "exac_maf":"-:0.293,-:8.261e-06"`, which would break parsing into Hail using the schema where those fields are specified as floats. 
+
+3. After all is set up, you can start a Dataproc cluster passing the initialization script explicitly with `init`, instead of using the `vep` parameter. See example in [test/test-dataproc-wrapper.py](test/test-dataproc-wrapper.py). Make sure you set larger resources for the workers (the highmem machine type and larger storage). Hail would do that automatically with `--vep`, but with a custom `--init` we have to do that explicitly.
+
+Within scripts that you submit to that cluster, you can call the `hl.vep` function with explicitly provided `config` parameter (otherwise it would attempt to get the VEP_CONFIG environment variable, which is only set with `--vep`). See example in [test/test-dataproc-script.py](test/test-dataproc-script.py).
+
+Note that VEP might print errors messages that it can't find `phylocsf_data` and `gerp_exons` tables in the loftee DB:
+
+```sh
+DBD::SQLite::db prepare failed: no such table: phylocsf_data at /root/micromamba/share/ensembl-vep-105.0-0/LoF.pm line 565, <$fh> line 158458.
+DBD::SQLite::db prepare failed: no such table: gerp_exons at /root/micromamba/share/ensembl-vep-105.0-0/gerp_dist.pl line 129, <$fh> line 158458.
+```
+
+It's expected as those tables are provided by `phylocsf_gerp.sql`, which are only provided for [GRCh37](https://personal.broadinstitute.org/konradk/loftee_data/GRCh37/), but not for [GRCh38](https://personal.broadinstitute.org/konradk/loftee_data/GRCh38/). See [this issue](https://github.com/konradjk/loftee/issues/39).
```

Comment:
just a note that https://github.com/populationgenomics/images/pull/44 (and the following PRs to fix it) solve this issue

Reply:
Yeah, I wanted to link that but forgot! Thanks a lot for solving that issue, that's a really nice one